portal_backend/scheme_utils/trusted_aps_import.py

```diff
@@ -248,7 +247,7 @@ def create_credential_attributes(
                 continue
 
             name = attr["Name"]
-            desc = attr.get("Description", {})
+            desc = attr.get("Description")
```

Comment:
I think the fallback value here is necessary to prevent possible errors on line 259 and 260